Library1/Library.fs

```diff
@@ -22,9 +22,389 @@ module FetchData =
+            builder.OpenElement(20, "th")
+            builder.AddContent(21, "Date")
+            builder.CloseElement()
+            builder.AddContent(22, "\n                ")
+            builder.OpenElement(23, "th")
+            builder.AddContent(24, "Temp. (C)")
+            builder.CloseElement()
+            builder.AddContent(25, "\n                ")
+            builder.OpenElement(26, "th")
+            builder.AddContent(27, "Temp. (F)")
+            builder.CloseElement()
+            builder.AddContent(28, "\n                ")
+            builder.OpenElement(29, "th")
+            builder.AddContent(30, "Summary")
+            builder.CloseElement()
+            builder.AddContent(31, "\n            ")
+            builder.CloseElement()
+            builder.AddContent(32, "\n        ")
+            builder.CloseElement()
+            builder.AddContent(33, "\n        ")
+            builder.OpenElement(34, "tbody")
+            builder.AddContent(35, "\n")
+            for forecast in forecasts do
+                builder.AddContent(36, "                ")
+                builder.OpenElement(37, "tr")
+                builder.AddContent(38, "\n                    ")
+                builder.OpenElement(39, "td")
+                builder.AddContent(40, forecast.Date.ToShortDateString())
+                builder.CloseElement()
+                builder.AddContent(41, "\n                    ")
+                builder.OpenElement(42, "td")
+                builder.AddContent(43, forecast.TemperatureC)
+                builder.CloseElement()
+                builder.AddContent(44, "\n                    ")
+                builder.OpenElement(45, "td")
+                builder.AddContent(46, forecast.TemperatureF)
+                builder.CloseElement()
+                builder.AddContent(47, "\n                    ")
+                builder.OpenElement(48, "td")
+                builder.AddContent(49, forecast.Summary)
+                builder.CloseElement()
+                builder.AddContent(50, "\n                ")
+                builder.CloseElement()
+                builder.AddContent(51, "\n")
+            builder.AddContent(52, "        ")
+            builder.CloseElement()
+            builder.AddContent(53, "\n    ")
+            builder.CloseElement()
+            builder.AddContent(54, "\n")
+        builder.AddContent(55, "\n")
+
+    override this.OnInitAsync() =
+        async {
+            let! result = Async.AwaitTask <| Library1.FetchData.fetchForecasts(this.Http)
+            forecasts <- result
+        }
+        |> Async.StartAsTask :> System.Threading.Tasks.Task
+
+    // NOTE: F# does not appear to support the global attribute target.
+    // See https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/attributes#remarks
+    [<global::Microsoft.AspNetCore.Blazor.Components.InjectAttribute>]
+    member val private Http : HttpClient = Unchecked.defaultof<HttpClient> with get, set

```

Comment:
The only piece I couldn't convert in the `FetchData` page was the `[<global::Microsoft.AspNetCore.Blazor.Components.InjectAttribute>]` because `global` does not appear to be a valid attribute target in F#. Is that right, @dsyme, @cartermp, @KevinRansom, or @TIHan? If not, how do I convert this attribute?